packages/kbn-reporting/common/constants.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const CONTENT_TYPE_CSV = 'text/csv';
```

Comment:
These should be removed from `x-pack/plugins/reporting/common/constants/index.ts` as well, so we don't have duplicates.

Reply:
I think I have it in the correct packages now to avoid duplicates and circular dependencies

Reply:
Hitting errors Ive gotten in the past with circular dependencies in commit [4c90245](https://github.com/elastic/kibana/pull/151801/commits/4c90245a65b60187669d4c3aca1780c82c413ae8)

Reply:
Actually this might be an issue with the import from elastic/elasticsearch 

import { errors as esErrors, estypes } from '@elastic/elasticsearch';  in 
`packages/kbn-generate-csv/src/generate_csv.ts`

Reply:
Ok root cause is the get_csv_panel_action.tsx file. The generate-csv package cannot be imported into this public file (node scripts/build_kibana_platform_plugins.js fails). The only variable impacted in this file is CSV_REPORTING_ACTION. 
In commit [de4dacf](https://github.com/elastic/kibana/pull/151801/commits/de4dacf926ab40805ee972f1b68d11e8bc5d600c), I made a separate package kbn/generate-csv-types to avoid this issue. After debating, creating a separate package seemed overkill, instead in commit [69589f0](https://github.com/elastic/kibana/pull/151801/commits/69589f0ecc6e4e4b1990042cff040922e45f382d) I moved the variable to the reporting-common. This should also pass the CI but not require an entirely new package.